src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteWaypointModel.js

```diff
@@ -142,32 +127,40 @@ export default class StandardRouteWaypointModel extends BaseModel {
         this._positionModel = null;
 
         /**
-         * Distance in nm from the previous waypoint.
+         * Any restrictions for a given fix
          *
-         * This property is set exterally by the `StandardRouteModel` and used only when called via
-         * `ArrivalBase.preSpawn()`.
+         * ex:
+         * - "A80+|S210"
+         * - "A80-"
+         * - "S230"
          *
-         * This value is mutable and is not intended to be re-used after its initial use.
+         * using null here to match current api, if restrictions dont exist for a given waypoint
+         * the consumers are expecting this to be null.
          *
-         * @property distanceFromPreviousWaypoint
-         * @type {number}
-         * @default -1
+         * @property _restrictions
+         * @type {string|null}
```

Comment:
This was actually not modified, just resorted. Your comment sheds some light on why it's like this:
```
 /**
         * Any restrictions for a given fix
         *
         * ex:
         * - "A80+|S210"
         * - "A80-"
         * - "S230"
         *
         * using null here to match current api, if restrictions dont exist for a given waypoint
         * the consumers are expecting this to be null.
         *
         * @property _restrictions
         * @type {string|null}
         * @default null
         * @private
         */
        this._restrictions = null;
```